mods/ts/mod.yaml

```diff
@@ -270,12 +271,21 @@ ModContent:
 			Download: basefiles
 		music: Base Game Music
 			TestFiles: ^Content/ts/scores.mix
-			Sources: tibsun, tibsun-linux, tfd, origin
+			Sources: tibsun, tibsun-linux, tfd, origin, firestorm
 			Download: music
+		expansion: Firestorm Expansion
+			TestFiles: ^Content/ts/expand01.mix
+			Sources: firestorm, tfd
+			Required: true
```

Comment:
In any case we need to resolve this before we can merge any of these prs.  One of the major goals and successes of the asset installer rework was that all of our different installation methods give identical results, so that we don't have to worry about horrible workarounds.